app/src/main/res/drawable/ic_event.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
This looks like a transparent color we can use @android:color/transparent, same for the size, 24dp should be in dimens.xml